src/screens/planActivity/TaskTable.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { SFC } from 'react';
+import { StyleSheet, View } from 'react-native';
+
+import { FullScreenTemplate } from 'components';
+import { dimensions, getElevation, palette } from 'styles';
+import { TableRow } from './TableRow';
+
+export const CELL_DIMENSIONS = {
```

Comment:
How do this fixed sizes stretch on bigger screens?

Reply:
I checked it out on tablet with screen 10'. Below is the screen from that device.
<img width="882" alt="Screenshot 2019-10-08 at 11 33 07" src="https://user-images.githubusercontent.com/45594475/66384713-8cb1d200-e9bf-11e9-893d-afaa5106ce0b.png">